src/scripts/scaling/scalingAction.sh

```diff
@@ -2,24 +2,272 @@
+function logScalingParameters() {
+  echo "scaling_action: $scaling_action" >> scalingAction.log
+  echo "wls_domain_uid: $wls_domain_uid" >> scalingAction.log
+  echo "wls_cluster_name: $wls_cluster_name" >> scalingAction.log
+  echo "wls_domain_namespace: $wls_domain_namespace" >> scalingAction.log
+  echo "operator_service_name: $operator_service_name" >> scalingAction.log
+  echo "operator_service_account: $operator_service_account" >> scalingAction.log
+  echo "operator_namespace: $operator_namespace" >> scalingAction.log
+  echo "scaling_size: $scaling_size" >> scalingAction.log
+}
+
+# Query WebLogic Operator Service Port
+function get_operator_internal_rest_port() {
+  local STATUS=$(curl -v --cacert /var/run/secrets/kubernetes.io/serviceaccount/ca.crt -H "Authorization: Bearer $(cat /var/run/secrets/kubernetes.io/serviceaccount/token)" -X GET $kubernetes_master/api/v1/namespaces/$operator_namespace/services/$operator_service_name/status)
+  if [ $? -ne 0 ]
+  then
+    echo "$currdate Failed to retrieve status of $operator_service_name in name space: $operator_namespace" >> scalingAction.log
+    echo "$currdate STATUS: $STATUS" >> scalingAction.log
+    exit 1
+  fi
+
+  local port
+  if [ -x "$(command -v jq)" ]; then
+    local extractPortCmd="(.spec.ports[] | select (.name == \"rest\") | .port)"
+    port=$(echo "${STATUS}" | jq "${extractPortCmd}")
+  else
+cat > cmds-$$.py << INPUT
+import sys, json
+for i in json.load(sys.stdin)["spec"]["ports"]:
+  if i["name"] == "rest":
+    print(i["port"])
+INPUT
+port=$(echo "${STATUS}" | python cmds-$$.py)
+  fi
+  echo "$port"
+}
+
+# Retrieve the api version of the deployed Custom Resource Domain
+function get_domain_api_version() {
+  # Retrieve Custom Resource Definition for WebLogic domain
+  local CRD=`curl -v --cacert /var/run/secrets/kubernetes.io/serviceaccount/ca.crt -H "Authorization: Bearer $(cat /var/run/secrets/kubernetes.io/serviceaccount/token)" -X GET $kubernetes_master/apis/apiextensions.k8s.io/v1beta1/customresourcedefinitions/domains.weblogic.oracle`
+  if [ $? -ne 0 ]
+    then
+      echo "$currdate Failed to retrieve Custom Resource Definition for WebLogic domain" >> scalingAction.log
+      echo "$currdate CRD: $CRD" >> scalingAction.log
+      exit 1
+  fi
+
+# Find domain version
+  local domain_api_version
+  if [ -x "$(command -v jq)" ]; then
+    domain_api_version=$(echo "${CRD}" | jq -r '.spec.version')
+  else
+cat > cmds-$$.py << INPUT
+import sys, json
+print(json.load(sys.stdin)["spec"]["version"])
+INPUT
+domain_api_version=`echo ${CRD} | python cmds-$$.py`
+  fi
+  echo "$domain_api_version"
+}
+
+# Retrieve Custom Resource Domain
+function get_custom_resource_domain() {
+  local DOMAIN=`curl -v --cacert /var/run/secrets/kubernetes.io/serviceaccount/ca.crt -H "Authorization: Bearer $(cat /var/run/secrets/kubernetes.io/serviceaccount/token)" $kubernetes_master/apis/weblogic.oracle/$domain_api_version/namespaces/$wls_domain_namespace/domains/$domain_uid`
```

Comment:
This is one of a few examples of long curl commands in this pull that is on a single line.  Can this line and the others be broken up into multiple lines in order to aid readability?